editions/tw5.com/tiddlers/system/Sources.tid

```diff
@@ -32,6 +45,7 @@ A static HTML representation of this tiddler is available at the URL:
 
 Help us to improve the documentation by sending a ~GitHub pull request for this tiddler:
 
+* [[Contribute to TW documentation using TiddlyWiki|https://saqimtiaz.github.io/tw5-docs-pr-maker/]]
```

Comment:
Rather than having this link as a bullet point, I think it might be better as part of the line above:

```
 Help us to improve the documentation by suggesting changes to this tiddler using the [[TiddlyWiki Docs PR Maker|https://saqimtiaz.github.io/tw5-docs-pr-maker/]]:
```